pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -616,6 +616,8 @@ func (m *kubeGenericRuntimeManager) SyncPod(pod *v1.Pod, _ v1.PodStatus, podStat
 		if err != nil {
 			createSandboxResult.Fail(kubecontainer.ErrCreatePodSandbox, msg)
 			glog.Errorf("createPodSandbox for pod %q failed: %v", format.Pod(pod), err)
+			ref, _ := ref.GetReference(api.Scheme, pod)
```

Comment:
At 550 line if get reference failed output error log, I think here may also be able to output the error log.